src/patternfly/components/JumpLinks/jump-links.scss

```diff
@@ -0,0 +1,169 @@
+  --pf-c-jump-links__link--before--BorderBottomWidth: 0;
+  --pf-c-jump-links__link--before--BorderLeftWidth: 0;
+  --pf-c-jump-links__link--before--BorderColor: transparent;
+  --pf-c-jump-links__link--focus--before--BorderTopWidth: var(--pf-global--BorderWidth--lg);
+  --pf-c-jump-links__link--focus--before--BorderLeftWidth: 0;
+  --pf-c-jump-links__link--focus--before--BorderColor: var(--pf-global--primary-color--100);
+  --pf-c-jump-links__item--m-current__link--before--BorderTopWidth: var(--pf-global--BorderWidth--lg);
+  --pf-c-jump-links__item--m-current__link--before--BorderLeftWidth: 0;
+  --pf-c-jump-links__item--m-current__link--before--BorderColor: var(--pf-global--primary-color--100);
+  --pf-c-jump-links--m-vertical__link--focus--before--BorderTopWidth: 0;
+  --pf-c-jump-links--m-vertical__link--focus--before--BorderLeftWidth: var(--pf-global--BorderWidth--lg);
+  --pf-c-jump-links--m-vertical__item--m-current__link--before--BorderTopWidth: 0;
+  --pf-c-jump-links--m-vertical__item--m-current__link--before--BorderLeftWidth: var(--pf-global--BorderWidth--lg);
+
+  // text
+  --pf-c-jump-links__link-text--Color: var(--pf-global--Color--200);
+  --pf-c-jump-links__link--hover__link-text--Color: var(--pf-global--Color--100);
+  --pf-c-jump-links__link--focus__link-text--Color: var(--pf-global--Color--100);
+  --pf-c-jump-links__item--m-current__link-text--Color: var(--pf-global--Color--100);
+
+  // label
+  --pf-c-jump-links__label--PaddingBottom: var(--pf-global--spacer--md);
+
+  display: flex;
+
+  &.pf-m-center {
+    justify-content: center;
+
+    .pf-c-jump-links__main {
+      align-items: center;
+    }
+  }
+
+  &.pf-m-vertical {
+    --pf-c-jump-links__list--PaddingTop: var(--pf-c-jump-links--m-vertical__list--PaddingTop);
+    --pf-c-jump-links__list--PaddingRight: var(--pf-c-jump-links--m-vertical__list--PaddingRight);
+    --pf-c-jump-links__list--PaddingBottom: var(--pf-c-jump-links--m-vertical__list--PaddingBottom);
+    --pf-c-jump-links__list--PaddingLeft: var(--pf-c-jump-links--m-vertical__list--PaddingLeft);
+    --pf-c-jump-links__list--before--BorderTopWidth: var(--pf-c-jump-links--m-vertical__list--before--BorderTopWidth);
+    --pf-c-jump-links__list--before--BorderLeftWidth: var(--pf-c-jump-links--m-vertical__list--before--BorderLeftWidth);
+    --pf-c-jump-links__link--focus--before--BorderTopWidth: var(--pf-c-jump-links--m-vertical__link--focus--before--BorderTopWidth);
+    --pf-c-jump-links__link--focus--before--BorderLeftWidth: var(--pf-c-jump-links--m-vertical__link--focus--before--BorderLeftWidth);
+    --pf-c-jump-links__item--m-current__link--before--BorderTopWidth: var(--pf-c-jump-links--m-vertical__item--m-current__link--before--BorderTopWidth);
+    --pf-c-jump-links__item--m-current__link--before--BorderLeftWidth: var(--pf-c-jump-links--m-vertical__item--m-current__link--before--BorderLeftWidth);
+    --pf-c-jump-links__list--FlexDirection: var(--pf-c-jump-links--m-vertical__list--FlexDirection);
```

Comment:
do you need this and the `flex-direction` declaration below?

Reply:
yeah, flex-direction: column needs to be set on the jump link and jump link list and it wasnt when i added the line on 78

Reply:
oh sorry, I thought they applied to the same element.